academy/4-my-own-chain/stored-game.md

```diff
@@ -289,6 +289,69 @@ Your stored game stores are only strings. But you know that they represent `sdk.
     )
     ```
 
+## Interact via the CLI
+
+Starport created a set of files for you. It is time to see whether you can already interact with your new checkers blockchain.
+
+1. Start the chain in its shell:
+
+    ```sh
+    $ starport chain serve --reset-once
+    ...
+    🌍 Tendermint node: http://0.0.0.0:26657
+    🌍 Blockchain API: http://0.0.0.0:1317
+    🌍 Token faucet: http://0.0.0.0:4500
+    ```
+
+2. Check the values saved in `NextGame`. Look at the relevant `client/cli` file Starport created to find out what command is relevant. Here it is [`query_next_game.go`](https://github.com/cosmos/b9-checkers-academy-draft/blob/3c69e22/x/checkers/client/cli/query_next_game.go#L14). You can instead also ask the CLI:
+
+    ```sh
+    $ checkersd query checkers --help
+    ```
+
+    Ok, that is [`show-next-game`](https://github.com/cosmos/b9-checkers-academy-draft/blob/3c69e2251f253288163021c75999709d8c25b402/x/checkers/client/cli/query_next_game.go#L14):
+
+    ```sh
+    $ checkersd query checkers show-next-game
+    NextGame:
+      creator: ""
+      idValue: "0"
+    ```
+
+    As expected.
+
+3. If you find the output suitable for reading on the spot but plan to use some scripts, get some information about your options:
```

Comment:
```suggestion
3. The `--output` flag allows you to get your results in a JSON format, which might be useful in case you would like to use a script to parse the information for example. When you use the `--help` flag, you see which flags are available for a specific command:
```